snapshot/snapshot.go

```diff
@@ -300,6 +300,11 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 			}
 
 			logCtx.Infof("found nydus meta layer id %s, parpare remote snapshot", id)
+
+			if err := o.fs.AddSnapshot(info.Labels); err != nil {
```

Comment:
Why only do `AddSnapshot` when the container running? If the user deletes the image before any container run, can nydus blobs be GC-ed?

Reply:
Okay I miss it, blob cache is generated at only container running.